superset/migrations/versions/b56500de1855_add_uuid_column_to_import_mixin.py

```diff
@@ -81,7 +81,8 @@ class ImportMixin:
 
 # Add uuids directly using built-in SQL uuid function
 add_uuids_by_dialect = {
-    MySQLDialect: """UPDATE %s SET uuid = UNHEX(REPLACE(uuid(), "-", ""));""",
+# TODO uuid from mysql aren't working as expected. that's why shouldn't be used for now. 
+#    MySQLDialect: """UPDATE %s SET uuid = UNHEX(REPLACE(uuid(), "-", ""));""",
```

Comment:
@iercan can you try changing it to:

```python
MySQLDialect: """UPDATE %s SET uuid = U UNHEX(REPLACE(CONVERT(UUID() using utf8mb4), '-', ''));"""
```

Based on https://dba.stackexchange.com/a/278540

Reply:
This solution worked. I will create another PR